### ISSUE TITLE
Windows Machine

### ISSUE THREAD

**Issue by duraes** [hicsail/ionic-hockeyapp#7]
Hi,

node v8.4.0
npm 5.3.0

tried on my windows machine, got this error:

```
λ npm install ionic-hockeyapp --save

> ionic-hockeyapp@0.1.3 postinstall d:\Sistemas\Globo\Samara.Mobile\TvGlobo.Samara.Mobile\node_modules\ionic-hockeyapp
> typings install

'typings' não é reconhecido como um comando interno
ou externo, um programa operável ou um arquivo em lotes.
npm WARN optional SKIPPING OPTIONAL DEPENDENCY: fsevents@1.1.2 (node_modules\fsevents):
npm WARN notsup SKIPPING OPTIONAL DEPENDENCY: Unsupported platform for fsevents@1.1.2: wanted {"os":"darwin","arch":"any"} (current: {"os":"win32","arch":"x64"})

npm ERR! code ELIFECYCLE
npm ERR! errno 1
npm ERR! ionic-hockeyapp@0.1.3 postinstall: `typings install`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the ionic-hockeyapp@0.1.3 postinstall script.
npm ERR! This is probably not a problem with npm. There is likely additional logging output above.
```

**Comment by gregfrasco** [comment]
@duraes  try `npm install typings` then `npm install ionic-hockeyapp --save`

**Comment by duraes** [comment]
Ok, that resolved.

But got this message
Typings is deprecated in favor of NPM @types -- see README for more information

**Comment by duraes** [comment]
Thanks!

**Comment by gregfrasco** [comment]
This issue has been resolved in v0.1.6. If still experience this issue on v0.1.6 or greater please create an issue.

**Comment by duraes** [comment]
Hi,

After last version, i got this error

```
            Cannot find name 'HockeyApp'.
         L37:    private hockeyapp: HockeyApp
      L38:  ) {

Error: Failed to transpile TypeScript
```

I think the missing typings is to blame, any chance to get type definitions on " @types / ionic-hockeyapp " ( weird issue formatting this )?

**Comment by gregfrasco** [comment]
Fix in v0.1.7
